mscore/startupWizard.cpp

```diff
@@ -84,6 +84,24 @@ StartupWizardPage1::StartupWizardPage1(QWidget* parent)
       setLayout(layout);
       }
 
+StartupWizardPage2::StartupWizardPage2(QWidget* parent)
```

Comment:
Do we really need a new page, can't it get combined with the existing one, which isn't exactly full?